src/webapp/topology.py

```diff
@@ -324,9 +321,8 @@ def get_tree(self, filters: Filters = None) -> MaybeOrderedDict:
             return
         # unlike the other filters, if past_days is not specified, _no_ past downtime is shown
         if filters.past_days >= 0:
-            # Filter out all downtimes that are older than 'past_days' so we swap its sign to account for end_age being
-            # negative for past end times. Always show downtimes with end_ages in the future.
-            if self.end_age.total_seconds() // 86400 < -filters.past_days:
+            # Filter out downtimes older than 'past_days'
+            if self.end_age.total_seconds() // 86400 > filters.past_days:
```

Comment:
I think it's worth keeping the comment about showing downtimes with `end_age` in the future to remind ourselves that this block doesn't just affect past downtimes.

Reply:
The block _only_ affects past downtimes. All present and future downtimes are shown.

Reply:
Ok, nevermind then!